internal/controller/apiextensions/claim/api_test.go

```diff
@@ -27,6 +27,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/crossplane/crossplane-runtime/pkg/meta"
+
```

Comment:
This whitespace should be removed so that all the runtime packages are grouped together.